core/autoscaling.go

```diff
@@ -8,6 +8,13 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/autoscaling"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"math"
+)
+
+const (
+	OnDemandPercentageLong  = "on_demand_percentage"
+	OnDemandNumberLong      = "on_demand_number"
```

Comment:
I think we should prefix these with something, to make it more clear what they are about:

I am thinking at something like `autospotting_on_demand_percentage` and `autospotting_on_demand_number`